Extractor/Zero3/Zero3_DirectoryTable.hpp

```diff
@@ -0,0 +1,59 @@
+
+#pragma once
+
+#include <vector>
+#include <string>
+
+const std::vector<std::string> ZERO_3_DIRECTORY_LIST = {
```

Comment:
No need of it being a vector because final array size is already known on compilation